src/native/corehost/corehost.proj

```diff
@@ -8,7 +8,7 @@
 
   <PropertyGroup>
     <IncrementalNativeBuild Condition="'$(IncrementalNativeBuild)' == ''">true</IncrementalNativeBuild>
-    <BuildCoreHostDependsOn>GetProductVersions;GenerateRuntimeVersionFile</BuildCoreHostDependsOn>
+    <BuildCoreHostDependsOn>GenerateRuntimeVersionFile</BuildCoreHostDependsOn>
```

Comment:
I checked and I couldn't find a place under src/native that still depended on the `GetProductVersions` target.